terraform/examples/daos_cluster/terraform.tfvars.perf.example

```diff
@@ -26,11 +26,19 @@ server_daos_scm_size       = 45
 # server_os_disk_type        = "pd-ssd"
 
 # pools = [
-#  {
-#    pool_name = "test_pool"
-#    pool_size = "1TB"
-#    containers = [ "test_container1", "test_container2" ]
-#  }
+#   {
+#     label = "test_pool"
+#     size = "1TB"
+#     tier_ratio = 3
+#     properties = "reclaim:disabled"
+#     containers = [
+#       {
+#         label = "test_cont",
+#         type = "POSIX",
+#         properties = "rf:0"
```

Comment:
`properties = "rf:0"`  Would it be possible to clarify why this is needed and being provided in all examples? what does it mean? Should people always provide this? or something else instead?

Reply:
rf:0 is the default replication factory.   R = Replication :  F = Factory :  0 = No replica. 

 @lsitkiew   I would rather not mix Pool type changes into this PR to move these conversations to GCPDASOS-83 work. 
Also rf:0 should be set as a default value and only users what want something else need to be exposed to it.

Reply:
Ok, I will move this changes to other PR.